### DISCUSSION TITLE
QUESTION: $TERMUX_APK_RELEASE typo?

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-packages#8556] [Q&A] [answered]
Why does $TERMUX_APK_RELEASE return F_DROID rather than F-DROID?

No reason to change the name of the service if it's a string value... could this have been a typo?

**Comment by agnostic-apollo** [comment, marked as answer]
Screaming snake case, consistent with other sources.

**Reply by nathaneltitane** [reply]
Sounds good - thanks
